assets/app.js

```diff
@@ -86,6 +86,17 @@
 /************************************************************************/
 /******/ ({
 
+/***/ "./app.css":
```

Comment:
This file is causing you conflicts because it should not be committed to GitHub. We don't commit our built assets to the repo - these change all the time and are built by a compiler/transpiler, not be a developers own coding.

Reply:
Removed this file from my master